docs/root/intro/arch_overview/matching/_include/simple.yaml

```diff
@@ -0,0 +1,67 @@
+static_resources:
+  listeners:
+  - address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 443
+    listener_filters:
+    filter_chains:
+      filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          stat_prefix: ingress_http
+          http_filters:
+          - name: with-matcher
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.common.matching.v3.ExtensionWithMatcher
+              extension_config:
+                  name: envoy.filters.http.fault
+                  typed_config:
+                    "@type": type.googleapis.com/envoy.extensions.filters.http.fault.v3.HTTPFault
+                    abort:
+                      http_status: 503
+                      percentage:
+                        numerator: 0
+                        denominator: HUNDRED
+                    delay:
+                      fixed_delay: 3s
+                      percentage:
+                        numerator: 0
+                        denominator: HUNDRED
+              matcher:
+                  matcher_tree:
+                      input:
+                          name: request-headers
+                          typed_config:
+                              "@type": type.googleapis.com/envoy.type.matcher.v3.HttpRequestHeaderMatchInput
+                              header_name: some-header
+                      exact_match_map:
+                          # Note this additional indirection; this is a workaround for Protobuf oneof limitations.
+                          map:
+                              skip_filter: # This is the header value we're trying to match against.
```

Comment:
It's a little confusing seeing both action name skip and the key here being `skip_filter`. Could the key (which is header value) be "some_value_to_match_on" or something like that?